File: Baekjoon/Class1/09.2741 N 찍기.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim();

const n = Number(input);
const nums = [];
for(let i = 1; i <= n; i++)
    nums.push(i);
console.log(nums.join("\n"));

File: Baekjoon/Class1/09.2741 N 찍기.py
import sys
input = sys.stdin.readline

n = int(input())
nums = []
for i in range(1, n + 1):
    nums.append(i)
print("\n".join(map(str, nums)))

'''
n = int(input())
print("\n".join((str(i) for i in range(1, n + 1))))
'''